packages/commonwealth/test/unit/server_controllers/server_groups_controller.spec.ts

```diff
@@ -172,7 +172,7 @@ describe('ServerGroupsController', () => {
   describe('#createGroup', async () => {
     const controller = createMockedGroupsController();
     const { user, chain, address } = createMockParams();
-    const result = await controller.createGroup({
+    const [result] = await controller.createGroup({
```

Comment:
Can we add an assertion that the analytics object contains the correct event data here?

It's one of the benefits of decoupling event creation from event publishing